includes/abstracts/calendar.php

```diff
@@ -570,11 +570,17 @@ public function set_time_format( $format = '' ) {
 	 */
 	public function set_datetime_separator( $separator = '' ) {
 
+		$separator_spacing = get_post_meta( $this->id, '_calendar_datetime_separator_spacing', true );
 		if ( empty( $separator ) ) {
 			$separator = get_post_meta( $this->id, '_calendar_datetime_separator', true );
 		}
 
-		$this->datetime_separator = esc_attr( $separator );
+		if ( empty( $separator_spacing ) ) {
+			$separator = '&nbsp;' . trim( $separator ) . '&nbsp;';
+		} else {
+			$separator = str_replace( ' ', '&nbsp;', $separator );
+		}
+		$this->datetime_separator = wp_kses_post( $separator );
```

Comment:
I think it's fine to not explicitly add support for HTML. You can use your judgement here- if we can support HTML relatively easily then I would say why not. If it looks like it may be more involved than we can add it as a feature update for the future.